packages/swapper/src/swappers/zrx/ZrxSwapper.ts

```diff
@@ -1,7 +1,145 @@
-import { Swapper, SwapperType } from '../../api'
-
+import BigNumber from 'bignumber.js'
+import { ChainTypes } from '../../../../asset-service/dist'
+import { GetQuoteInput, Quote, QuoteResponse, Swapper, SwapperType } from '../../api'
+import { APPROVAL_GAS_LIMIT, DEFAULT_SOURCE, MAX_ZRX_TRADE } from '../../utils/constants'
+import { AxiosResponse } from 'axios'
+import { zrxService } from './utils'
+import { SwapSource } from '../..'
+export class ZrxError extends Error {
+  constructor(message: string) {
+    super(message)
+    this.message = `ZrxError:${message}`
+  }
+}
 export class ZrxSwapper implements Swapper {
   getType() {
     return SwapperType.Zrx
   }
+
+  private normalizeAmount(amount: string | undefined): string | undefined {
+    if (!amount) return undefined
+    return new BigNumber(amount).toNumber().toLocaleString('fullwide', { useGrouping: false })
+  }
+
+  /**
+   * Get a basic quote (rate) for the pair
+   * @param input
+   */
+  async getQuote(input: GetQuoteInput): Promise<Quote | undefined> {
+    const {
```

Comment:
Im wondering if it would make sense to pull these methods out of the ZrxSwapper and have them in their own files.

My thought is that this file is going to get huge and it might be helpful to have them broken down into more digestible chucks and that way the test file wont get huge either.

thinking then all of our methods in this zrxSwapper would look more like this

```
async getQuote(input: GetQuoteInput): Promise<Quote | undefined> {
  return await importedGetQuote.bind(this, input)
}
```